go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/kubernetes-sigs/ingress2gateway
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```

Comment:
not sure we wants this line.
![image](https://github.com/kubernetes-sigs/ingress2gateway/assets/76940994/f7ad9bda-3d4c-4d5d-8b29-68d144952bb4)

Reply:
It was automatically added after bumping to 1.21. Removed 👍